app/assets/stylesheets/_base.scss

```diff
@@ -16,3 +16,24 @@ hr{
   border: 1px solid #A2A2A2;
   margin-bottom: 20px;
 }
+
+.beta-feedback{
+  line-height: 32px;
+}
+
+.app-title{
+  font-family: "Georgia";
+  line-height: 56px;
```

Comment:
Properties should be ordered font-family, font-size, line-height, margin-bottom, margin-left, margin-top